enatega-multivendor-admin/src/views/Dispatch.jsx

```
@@ -272,18 +293,17 @@ const Orders = props => {
 
   const filtered =
     searchQuery.length < 3
-      ? dataOrders && dataOrders.getActiveOrders
+      ? dataOrders && dataOrders.getActiveOrdersWithPagination.orders
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)